pipeline_dp/dp_computations.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 import pipeline_dp
-
+from pipeline_dp.aggregate_params import NoiseKind
```

Comment:
Nit:[ import only modules](https://google.github.io/styleguide/pyguide.html#22-imports)  (it's fine to import classes in tests)

Reply:
Will address in separate PR 👍